src/controllers/surveys.cr

```diff
@@ -19,7 +19,7 @@ class Surveys < Application
   # returns a list of surveys
   @[AC::Route::GET("/")]
   def index : Array(Survey::Responder)
-    Survey.query.select("id, title, description, trigger, zone_id, pages").to_a.map(&.as_json)
+    Survey.query.select("id, title, description, trigger, zone_id, building_id, pages").to_a.map(&.as_json)
```

Comment:
Should we make this filterable?
So you can filter the surveys by zone_id or building_id?

Reply:
done